### ISSUE TITLE
pydot.Dot.create(pydot.Dot()) returns assertion error

### ISSUE THREAD

**Issue by pj1989** [pydot/pydot#222]
The command
```
pydot.Dot.create(pydot.Dot())
```
returns 
```
"dot" with args ['-Tps', 'C:\\Users\\LUCA~1.AME\\AppData\\Local\\Temp\\tmp9jsxm52p'] returned code: 1

stdout, stderr:
 b''
b''

Traceback (most recent call last):

  File "<ipython-input-11-2d049bb9ae08>", line 1, in <module>
    pydot.Dot.create(pydot.Dot())

  File "C:\Users\Luca.Amerio\AppData\Local\Continuum\anaconda3\lib\site-packages\pydot.py", line 1945, in create
    assert process.returncode == 0, process.returncode

AssertionError: 1
```

Running `pydot` version 1.4.1

**Comment by pj1989** [comment]
The problem can be solved commenting line 117 
```
program += extension
```
in `pydot.py`.

In other words, the subprocess call does not need to include the program extension

**Comment by peternowee** [comment]
Judging by the discussion in the keras-issue you link to, which mentions `dot.bat` and `dot.exe`, and by the nature of your fix, I think you ran into issue pydot/pydot#205. I will mention your fix there. If you agree, you can close this issue and continue discussion in pydot/pydot#205.

**Comment by peternowee** [comment]
As mentioned in my previous comment, I am closing this as a suspected duplicate of pydot/pydot#205.
